### PR TITLE
Jof guide wide minor edits

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**
Guide wide edits, couple of qs

### CODE REVIEW

File: doc-Service-Telemetry-Framework/modules/con_installation-size.adoc [hunk 1]
```diff
@@ -32,27 +32,34 @@ The size of your {OpenShift} installation depends on the following factors:
...
 ----
 
-If you have an existing environment it must be destroyed and recreated in order for the resource requests to have an effect. To recreate the environment first run `crc delete` to delete the virtual machine, and then run the provided command to create your environment.
+If you have an existing environment, delete it, and recreate it to ensure that the resource requests have an effect.
+
+To recreate the environment:
+
+. Run the `crc delete` command.
+
+. Run the provided command to create your environment.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/con_installation-size.adoc:63]: @leifmadsen Is the provided command "crc start --memory=49152 --cpus=8"?

File: doc-Service-Telemetry-Framework/modules/proc_configuring-red-hat-openstack-platform-overcloud-for-stf.adoc [hunk 2]
```diff
@@ -25,7 +25,8 @@
 
 To configure the {OpenStack} overcloud, you must configure collectd and {MessageBus}:
 
-. Create a file and name it `stf-connectors.yaml`. Save the file in the `/home/stack` directory.
+. Create a file and name it `stf-connectors.yaml`.
+. Save the file in the `/home/stack` directory.
 . Retrieve the QDR route address for the `MetricsQdrConnectors` host parameter. For more information, see Retrieving the QDR route address.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_configuring-red-hat-openstack-platform-overcloud-for-stf.adoc:30]: @mrunge @leifmadsen Need a section on Retrieving the QDR route address. Any volunteers? :)

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 3]
```diff
@@ -319,7 +324,8 @@ localhost                  : ok=37   changed=0    unreachable=0    failed=0    s
 
 . View the pods and the status of each pod to determine that all workloads are operating nominally:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc:325]: @leifmadsen This line is duplicated, I possibly did this in another PR. Not sure how to retrieve the original line!

File: doc-Service-Telemetry-Framework/modules/proc_validating-clientside-installation.adoc [hunk 4]
```diff
@@ -21,6 +21,6 @@
 // Start the title with a verb, such as Creating or Create. See also
 // _Wording of headings_ in _The IBM Style Guide_.
 [id="validating-clientside-installation_{context}"]
-= Validating clientside installation
+= Validating client-side installation
 
-Ensure that collectd is connected to QDR and QDR connected to STF. Use qdstat to validate this. Can also refer to AMQ Interconnect documentation on how to manage AMQ via qdstat.
+Ensure that collectd is connected to QDR and that QDR is connected to STF. Use qdstat to validate this. Can also refer to AMQ Interconnect documentation on how to manage AMQ via qdstat.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_validating-clientside-installation.adoc:26]: @mrunge @leifmadsen This is another section that we need to complete. Any takers?

File: doc-Service-Telemetry-Framework/modules/con_installation-size.adoc (hunk 1 above)
Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/con_installation-size.adoc:63]: Yep

File: doc-Service-Telemetry-Framework/modules/proc_configuring-red-hat-openstack-platform-overcloud-for-stf.adoc (hunk 2 above)
Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_configuring-red-hat-openstack-platform-overcloud-for-stf.adoc:30]: @JoanneOFlynn2018 assigned to me, I will get it today. Basically just going to be `oc get routes`.

File: doc-Service-Telemetry-Framework/modules/con_installation-size.adoc [hunk 5]
```diff
@@ -59,7 +59,14 @@ If you have an existing environment, delete it, and recreate it to ensure that t
 To recreate the environment:
 
 . Run the `crc delete` command.
++
+----
+crc delete
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/con_installation-size.adoc:64]: @leifmadsen Is the command "crc delete <crc-enviroment-name>" where you replace <crc-enviroment-name> with the actual name?

File: doc-Service-Telemetry-Framework/modules/proc_validating-clientside-installation.adoc (hunk 4 above)
Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_validating-clientside-installation.adoc:26]: @JoanneOFlynn2018 yea I'm working on it :) Assigned --> Leif

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc (hunk 3 above)
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc:325]: @leifmadsen Yeah they are still the same here. Looks like the first instance of "View the pods" is the step that we should change. 
For the first "view the pods" line, is that output as a result of running the "oc logs" command in the previous step?

Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc:325]: Oh and thanks for "git blame"

File: doc-Service-Telemetry-Framework/modules/proc_configuring-red-hat-openstack-platform-overcloud-for-stf.adoc (hunk 2 above)
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_configuring-red-hat-openstack-platform-overcloud-for-stf.adoc]: Added in last commit

**Pull request merged**